st2common/st2common/constants/triggers.py

```diff
@@ -100,6 +101,23 @@
     }
 }
 
+INQUIRY_TRIGGER = {
+    'name': 'st2.generic.inquiry',
+    'pack': SYSTEM_PACK_NAME,
+    'description': 'Trigger indicating a new "inquiry" has entered "pending" status',
+    'payload_schema': {
+        'type': 'object',
+        'properties': {
+            'liveaction_id': {},
```

Comment:
Added more detail in c1c93da30. Also using `id` instead of `liveaction_id` to keep confusion down if we ever create a proper data model for inquiries